content/ko/community/_index.html

```diff
@@ -0,0 +1,236 @@
+---
+title: 커뮤니티
+layout: basic
+cid: community
+---
+
+<div class="newcommunitywrapper">
+<div class="banner1">
+  <img src="/images/community/kubernetes-community-final-02.jpg" alt="Kubernetes Conference Gallery" style="width:100%;padding-left:0px" class="desktop">
+  <img src="/images/community/kubernetes-community-02-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%;padding-left:0px" class="mobile">
```

Comment:
nit: You might want to localize the `alt` attribute here.

Reply:
I found some things that needed to be modified and solved them. Including your opinion, I solved it.